### PR TITLE
Chapter 7: Sign Up 

### PR DESCRIPTION
**Pull request by mohdzakwan99**
Ready

### CODE REVIEW

**Comment by mohdzakwan99** [comment]
ready

File: app/assets/stylesheets/users.scss [hunk 1]
```diff
@@ -0,0 +1,3 @@
+// Place all the styles related to the Users controller here.
+// They will automatically be included in application.css.
```
Comment (ngocvt-0484) [inline app/assets/stylesheets/users.scss:2]: delete this file because not use

File: app/controllers/users_controller.rb [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+    end
+  end
+
+  private
+
+  def user_params
+    params.require(:user).permit :name, :email, :password, :password_confirmation
+  end
+
+  def load_user
```
Comment (ngocvt-0484) [inline app/controllers/users_controller.rb:26]: you defined this function but not call anywhere

you can call it by before action:
```
  before_action :load_user, only: :show 
```
and remove line 3
```
   @user = User.find_by id: params[:id]
```

File: app/helpers/users_helper.rb [hunk 3]
```diff
@@ -0,0 +1,8 @@
+module UsersHelper
+  # Returns the Gravatar for the given user.
+  def gravatar_for user
+    gravatar_id = Digest::MD5::hexdigest user.email.downcase
+  gravatar_url = "https://secure.gravatar.com/avatar/#{gravatar_id}"
```
Comment (ngocvt-0484) [inline app/helpers/users_helper.rb:5]: wrong indent

File: app/views/users/new.html.erb [hunk 4]
```diff
@@ -0,0 +1,33 @@
+<% provide :title, t("sign_up") %>
+<h1>Sign up</h1>
+<div class="row">
+  <div class="col-md-6 col-md-offset-3">
+    <%= form_for @user, url: signup_path do |f| %>
+
+  <% if @user.errors.any? %>
```
Comment (ngocvt-0484) [inline app/views/users/new.html.erb:7]: wrong indent

File: test/controllers/users_controller_test.rb [hunk 5]
```diff
@@ -0,0 +1,13 @@
+require "test_helper"
+
+class UsersControllerTest < ActionDispatch::IntegrationTest
+  test "should get show" do
```
Comment (ngocvt-0484) [inline test/controllers/users_controller_test.rb:4]: dont need do test, so please delete it

**Comment by mohdzakwan99** [comment]
ready

**Pull request closed without merging**